packages/angular/src/sdk.ts

```diff
@@ -0,0 +1,10 @@
+import { BrowserOptions, init as browserInit } from '@sentry/browser';
+
+/**
+ * Inits the Angular SDK
+ */
+export function init(options: BrowserOptions): void {
+  options.metadata = options.metadata || {};
+  options.metadata.name = options.metadata.name || 'sentry.javascript.angular';
```

Comment:
Shall we further namespace this into `options.metadata.sdk.name`? Group together SDK info.

Reply:
Every additional depth level require writing an additional presence check. I'd rather rename it to `sdkName` and `sdkVersion` instead if we'd want it to be more verbose.